bpf/lib/conntrack.h

```diff
@@ -754,6 +754,7 @@ static __always_inline int ct_create6(const void *map_main, const void *map_rela
 
 	entry.lb_loopback = ct_state->loopback;
 	entry.node_port = ct_state->node_port;
+	relax_verifier();
```

Comment:
Could you add a comment saying why it's needed?

Reply:
We don't have such comments for all the other points where we added a `relax_verifier()` pruning point (as the context for that is in the commit that introduced back this helper) so I'd rather not do that for consistency.

I could add some context on how I found these pruning points, but that would probably be too long (I have a WIP document where I describe the process which is already a few pages long and I'm working on some scripts to automate this a bit more)

Reply:
Could you then add a short explanation of your idea to the git commit message?

Reply:
Thanks!